catalog.html

```diff
@@ -0,0 +1,89 @@
+        </a>
+        <nav>
+          <ul class="header__nav-list">
+            <li>
+              <a href="index.html" class="header__navigation-link">Главная</a>
+            </li>
+            <li>
+              <a class="header__navigation-link header__navigation-link_active">Фотографии</a>
+            </li>
+            <li>
+              <a href="form.html" class="header__navigation-link">Конкурс</a>
+            </li>
+            <li>
+              <a href="" class="header__navigation-link">HTML Academy</a>
+            </li>
+          </ul>
+          <button class="header__menu-burger" aria-label="открыть меню"><svg class="header__menu-icon" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false"><path d="M1.137 0c-.15-.003-.3.021-.439.07C.558.12.432.195.326.288A.98.98 0 0 0 .08.614.877.877 0 0 0 0 1v2c0 .265.12.52.333.707.213.188.502.293.804.293h47.726c.15.003.3-.021.439-.07.14-.05.266-.123.372-.217a.98.98 0 0 0 .246-.327A.877.877 0 0 0 50 3V1c0-.265-.12-.52-.333-.707A1.22 1.22 0 0 0 48.863 0H1.137Zm0 10a1.25 1.25 0 0 0-.439.07c-.14.05-.266.123-.372.217a.98.98 0 0 0-.246.327A.877.877 0 0 0 0 11v2c0 .265.12.52.333.707.213.188.502.293.804.293h47.726c.15.003.3-.021.439-.07.14-.05.266-.123.372-.217a.98.98 0 0 0 .246-.327A.877.877 0 0 0 50 13v-2c0-.265-.12-.52-.333-.707a1.22 1.22 0 0 0-.804-.293H1.137Zm0 10c-.15-.003-.3.021-.439.07-.14.05-.266.123-.372.217a.98.98 0 0 0-.246.327A.877.877 0 0 0 0 21v2c0 .265.12.52.333.707.213.187.502.293.804.293h47.726c.15.003.3-.021.439-.07.14-.05.266-.124.372-.217a.98.98 0 0 0 .246-.327A.877.877 0 0 0 50 23v-2c0-.265-.12-.52-.333-.707a1.22 1.22 0 0 0-.804-.293H1.137Z" fill="#fff"/></svg></button>
+        </nav>
+      </div>
+    </header>
+    <main class="page__content">
+      <div class="intro intro_page_catalog">
+        <h1 class="intro__title intro__title_page_catalog">Фотографии пользователей</h1>
+      </div>
+    </main>
+    <footer class="footer page__footer">
+      <div class="footer__container">
+        <a class="logo2" href="#">
```

Comment:
Thank you for your feedback! The same errors are present in form.html. Should I make changes there in this pull request, or is it better to create a new one

Reply:
@NazarPauk, great catch! I agree that it's preferable to address those issues in a separate pull request since this one is mainly focused on creating a new page. Let's tackle it in a new branch where we can (1) ensure footer logos direct to `index.html` and (2) update social link descriptions to maintain consistency across the app. I suggest going through both `index.html` and `form.html` 😉

Regarding the Pink logos on the homepage, we might not want to make header and footer logos to be actual links. It's important to consider that having an active link to any page on that page itself can confuse users and disrupt their navigation, as it simply refreshes the page.

One solution here could be removing the `href` attributes from the `logo` and `logo2` blocks on `index.html` and updating CSS selectors like
```css
.logo[href]:hover {
}

.logo[href]:active {
}
```
This syntax ensures styles only apply to elements with the href attribute present.

Please let me know once you have gone through this message so I can proceed with merging this PR. If you have any further questions, please feel free to ask ❤️

Reply:
Thank you for the idea and detailed description, I will start working on the [Issue](https://github.com/p-foundation/pink-app/issues/45) that addresses the problem with descriptions and their inconsistency with the website language. I will also create a separate branch to remove links from the logos on the homepage.